webapp/routes.go

```diff
@@ -37,8 +37,8 @@ func RegisterRoutes() {
 	shared.AddRoute("/runs", "test-runs", testRunsHandler)
 	shared.AddRoute("/test-runs", "test-runs", testRunsHandler) // Legacy name
 
-	// Dashboard for the interop effort, by year.
-	shared.AddRoute("/{name:(?:compat|interop-)}{year:[0-9]+}", "interop-dashboard", interopHandler)
+	// Dashboard for the interop effort, by year. Defaults to current year.
+	shared.AddRoute("/{name:(?:compat|interop-?)}{year:[0-9]*}", "interop-dashboard", interopHandler)
```

Comment:
/interop2023 will be a legit URL now. Would that be an issue for the endpoint? Do we parse the "-"?

Reply:
I was thinking this was fine, as the site will still function and redirect as intended in this case, but there are also instances of odd behavior (e.g. `/interop-`, `/compat`, and `/compat-` all redirect to the current dashboard now).

Based on jgraham's comment, I think it might be best to separate the regex out, as it's becoming a somewhat complex statement, and it's possible that `/interop` will one day be more of a landing page that explains the Interop effort as a whole, rather than just navigating to the dashboard.

Reply:
Yea I agree. Let's separate it out. Thanks for going back and forth!